CombinatorialGames/Surreal/Division.lean

```diff
@@ -146,42 +156,42 @@ lemma option_mul_inv_lt {x : IGame} [Numeric x] (hx : 0 < x)
     (hl' : ∀ y ∈ x.leftMoves, 0 < y → y * y⁻¹ ≈ 1) (hr' : ∀ y ∈ x.rightMoves, y * y⁻¹ ≈ 1) :
     (∀ y ∈ (x * x⁻¹).leftMoves, y < 1) ∧ (∀ y ∈ (x * x⁻¹).rightMoves, 1 < y) := by
   have := numeric_inv hx hl hr hl' hr'
-  obtain ⟨Hl, Hr⟩ := numeric_option_inv hl hr
+  obtain ⟨Hl, Hr⟩ := numeric_option_inv hx hl hr
   rw [forall_leftMoves_mul, forall_rightMoves_mul]
   refine ⟨⟨?_, ?_⟩, ⟨?_, ?_⟩⟩
   all_goals
-    intro y hy a ha
-    first | have := Numeric.of_mem_leftMoves hy | have := Numeric.of_mem_rightMoves hy
+    intro y hyx a ha
```

Comment:
This proof didn't really change that much, I just tried to enforce the naming convention that `hyx : y ∈ x.(left|right)Moves` and `hy : 0 < y` (since it's a bit weird to have `hx : 0 < x` and `hy' : 0 < y` otherwise)